### DISCUSSION TITLE
Closable Tab will be supported in the future?

### DISCUSSION THREAD

**Discussion by liuyang3688** [quasarframework/quasar#13476] [General - Components / Directives / etc] [answered]
In my project, i need a tab bar in which each tab item can be managed by open/close/fresh etc. operations. But, i couldnt find such comp supporting this behavior. 

What's your suggestion? Tell me at your free. Thanks.

**Comment by pratik227** [comment, marked as answer]
https://codepen.io/Pratik__007/pen/WNMOjwo?editors=101
